resolwe/flow/models.py

```diff
@@ -271,24 +270,20 @@ class Meta(BaseModel.Meta):
 
     """
 
+    requirements = JSONField(default=dict)
 
-def render_template(template_string, context):
-    """Render template based on Dango template language."""
-    template_headers = [
-        '{% load resource_filters %}',
-        '{% load process_fields %}',
-        '{% load mathfilters %}',
-    ]
 
-    custom_template_tags = getattr(settings, 'RESOLWE_CUSTOM_TEMPLATE_TAGS', [])
-    if not isinstance(custom_template_tags, list):
-        raise KeyError("`RESOLWE_CUSTOM_TEMPLATE_TAGS` setting must be a list.")
+def render_template(process, template_string, context):
+    """Render template using the specified expression engine."""
+    from .managers import manager
```

Comment:
@kostko: As discussed, please find out how Celery sends tasks to workers and report the findings here.